.docker/entrypoint/lib/commander.rb

```diff
@@ -50,51 +49,60 @@ def log(severity, message)
-        verify(jekyll_config) if verify
+        ignore_urls = args['--ignore-url']
+        log_level = args['--log-level']
+        jekyll_config_provider = JekyllConfigProvider.new(@jekyll_env, log_level)
+        jekyll_config = jekyll_config_provider.provide(command)
+        execute_command(jekyll_config, command, dry_run, verify, log_level)
+        verify(jekyll_config, ignore_urls, log_level) if verify
       end
 
-      def execute_command(jekyll_config, command, dry_run, verify)
+      def find_command(args)
+        return 'build' if args['build'] == true
+        return 'serve' if args['serve'] == true
+        return 'deploy' if args['deploy'] == true
+
+        raise CommandLineArgumentError, 'Unkonw command'
+      end
+
+      def execute_command(jekyll_config, command, dry_run, verify, log_level)
         case command
         when 'deploy'
           deploy(jekyll_config, verify, dry_run)
-        when 'build', 'serve'
-          jekyll_command(jekyll_config, command, dry_run)
+        when 'build'
+          build(jekyll_config, dry_run, log_level)
+        when 'serve'
+          serve(jekyll_config, dry_run, log_level)
         else
           raise CommandLineArgumentError, "Unknown command '#{command}'"
         end
       end
 
-      def verify(jekyll_config)
-        verifier = provide_instance(:verify, jekyll_config)
-        verifier.verify
+      def verify(jekyll_config, ignore_urls, log_level)
+        verifier = @commands.verifier.new(jekyll_config, log_level)
+        verifier.verify(ignore_urls)
       end
 
       def deploy(jekyll_config, dry_run, verify)
-        deployer = provide_instance(:deploy, jekyll_config, @jekyll_var_dir)
+        deployer = @commands.deployer.new(jekyll_config, @jekyll_env.var_dir)
         deployer.deploy(dry_run, verify)
       end
 
-      def jekyll_command(jekyll_config, command, dry_run)
-        log(:warn, "Warning: --dry-run has no effect on the `jekyll #{command}` command.") if dry_run
+      def build(jekyll_config, dry_run, log_level)
+        log(:warn, 'Warning: --dry-run has no effect on the `jekyll build` command.') if dry_run
 
-        jekyll_commander = provide_instance(command, jekyll_config)
-        # jekyll_commander = Jekyll::PlantUml::Commands::JekyllCommander.new(jekyll_config)
-        jekyll_commander.execute(command)
+        jekyll_builder = @commands.builder.new(jekyll_config, log_level)
+        jekyll_builder.execute
       end
 
-      def provide_instance(command, *args)
-        command_symbol = command.to_sym
-        class_definition = @commands[command_symbol]
-        raise ArgumentError, "No class definition found for command '#{command}'" if class_definition.nil?
-
-        return class_definition unless class_definition.is_a? Class
+      def serve(jekyll_config, dry_run, log_level)
+        log(:warn, 'Warning: --dry-run has no effect on the `jekyll serve` command.') if dry_run
```

Comment:
Codacy found an issue: [Modifier form of `if` makes the line too long.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)